packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md

```diff
@@ -0,0 +1,168 @@
+---
+id: Release highlights
+title: Release highlights
+section: get-started
+---
+
+import './get-started.css';
+
+# What's new with PatternFly 5.0 
+
+The following sections highlight the new features and updates included in this release. You can [view the full details for this release on GitHub](https://github.com/patternfly/patternfly-react/releases) and [refer to our upgrade guide](/get-started/upgrade) to update from PatternFly 4 to PatternFly 5.
+
+## Dark theme support
+
+While PatternFly has supported a dark theme in beta for some time, our dark theme has not been fully documented or tested. As part of the release of PatternFly 5, we were dedicated to thoroughly testing our beta implementation to ensure that it was ready for everyone. With this major release, we're excited to say that PatternFly's dark theme is fully vetted and supported. You can configure your applications' dark theme to update based on users’ system preferences.
+
+We are also currently working on introducing more robust theming capabilities in a future PatternFly major release. Going forward, all future themes will include a light and dark theme.
+
+There are a couple of ways to enable dark theme in your products. You can: 
+
+1. Add `patternfly-theme-dark.css` to a page and add the `.pf-theme-dark` class to the page's `<html>` element, or 
+2. Apply the `.pf-theme-dark` class to the browser's `prefers-color-scheme` media query. 
```

Comment:
These steps still need to be updated, not sure if that is known? Wanted to mention just in case. I can help with that if needed. The TL;DR is there is really only one way to enable dark theme - add the `.pf-v5-theme-dark` class to the `<html>` element.